SWIG/timeseries.i

```diff
@@ -29,6 +29,12 @@ using QuantLib::TimeSeries;
 using QuantLib::IntervalPrice;
 %}
 
+#if defined(JAVA_CLOSEABLE)
```

Comment:
I'd go for

    #if defined (SWIGJAVA) && defined (JAVA_CLOSEABLE)

since one can configure for more than one language at the same time.